app/components/projects/createCardProject.js

```diff
@@ -0,0 +1,5 @@
+import CardProject from "./CardProject";
+
+export default function createCardProject(SERVICES) {
```

Comment:
why SERVICES and not services

Reply:
to be different and consistent. All the arrays' names in this project are in uppercase
 SERVICES is an array and in my opinion, it is more clear since the word 'services' is used several times

Reply:
yes, but

https://google.github.io/styleguide/jsguide.html#naming-constant-names

https://google.github.io/styleguide/jsguide.html#naming-parameter-names

the array, which is a const, is correctly named using all caps

function parameters should be normal camel case

Reply:
also this: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#general_guidelines_for_javascript_code_examples

dont waste too much time on all these resources, just keep it mind that there are some standards most people follow. to break them we need a very good and clear reason